### ISSUE TITLE
Undefined variable: beforeFormView when adding wrapper layout

### ISSUE THREAD

**Issue by michaelgatuma** [tanthammar/tall-forms#39]
**Describe the bug**
I was following your documentation wiki for installing tall-forms and on the wrapper-layout step, I came into a problem that I am unable to solve. I am farely new to laravel so if there is something I skipped please be kind enough to help.
**Screenshots**
![image](https://user-images.githubusercontent.com/50529359/97108874-54bc2080-16e1-11eb-90e4-1b0b69a4cbb6.png)

**Environment:**
 - Tall-forms version: ^5.0
 - Livewire version: ^2.0
 - Laravel version: ^8.0
 - AlpineJS version (only needed if problem is Alpine related):
 - $wrapWithLayout true/false: Not Sure where to set this.
    - if true, Are you using your own wrapper component or the default one?: the default one
 - Optional model binding? no
 - Have you published the view files?: no
 - Have you published, and edited the config file?: yes published but not edited.
 - Is it a Create or Update form?: not applicable

**Fields Definition**
Please paste the code for your fields that is related to this issue.

**Comment by tanthammar** [comment]
Hello @MichaelGatuma 

I am unable to support you without more information. Need to see more code...
- Can you please describe more exactly how you setup your wrapper?
- Did you create `resources/views/components/pages/default.blade.php`?
- Does your app.blade.php have a {{ $slot }}?
- You need to have Livewire at least 2.5.6

**Or** set your form component `$wrapWithView` to `false`, that way your component will ignore the wrapper view.
You can continue with your installation and get back to that step later.

I made some changes to the "Wrapper" wiki page, hope it is easier to follow.

**Comment by tanthammar** [comment]
@MichaelGatuma 
I simplified the wrapper wiki page.
Please let me know how you are progressing with your installation.

**Comment by tanthammar** [comment]
@MichaelGatuma 
Didn't hear anything from you. Please reopen if issue remains.

**Comment by michaelgatuma** [comment]
> @MichaelGatuma
> I simplified the wrapper wiki page.
> Please let me know how you are progressing with your installation.

Hello @tanthammar ,
Sorry for the late reply. I was able to figure my way around tall-forms. I must say it makes things pretty easy. I successfully added  and tweaked the forms. However, the wiki is abit hard to follow for someone who is coming from laravel 7 and below and new to tall stack.

**Comment by tanthammar** [comment]
@MichaelGatuma 
Thanks for you reply.
I read the installations yesterday, again, and made som additional changes.
If I understand you correctly - you have installed it and you're up and running?

Please give me feedback on how I can change the installation tutorial to make it easier to follow.
It's hard sometimes when you know things by heart, to be objective and remember to describe every step correctly.

**Comment by michaelgatuma** [comment]
@tanthammar , this is not issues related. I just sponsored and I don't know how to get access to the sponsors repo.

**Comment by tanthammar** [comment]
@MichaelGatuma 
Fantastic Thank YOU! 
Means the world to me. 

I just got the sponsor email. I'll send you an invite to the sponsor repo and the discord chat as soon as I'm back at the office :)

**Comment by michaelgatuma** [comment]
Awesome, thanks 👌

**Comment by tanthammar** [comment]
Hi. I sent you an invite to the sponsors repo. Did you get it?

**Comment by tanthammar** [comment]
@MichaelGatuma Hi. Trying to send you an invite to the discord channel, but I can't find you there. What's your username?

**Comment by michaelgatuma** [comment]
@tanthammar I got your invitation thanks. My discord username is [gatuma]

**Comment by tanthammar** [comment]
@MichaelGatuma Trying to find you on discord... Are you on the Livewire channel?

**Comment by michaelgatuma** [comment]
> @MichaelGatuma Trying to find you on discord... Are you on the Livewire channel?

No not yet, let me join right away maybe it will make it easier.

**Comment by tanthammar** [comment]
@MichaelGatuma

Did you see that the invite link to discord is in the sponsors repo? 
In the Readme, front page.

**Comment by michaelgatuma** [comment]
Yeah sure @tanthammar . I have just joined
